src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -164,6 +164,65 @@ class SidebarLinks extends React.Component {
 SidebarLinks.propTypes = propTypes;
 SidebarLinks.defaultProps = defaultProps;
 
+/**
+ * This function (and the few below it), narrow down the data from Onyx to just the properties that we want to trigger a re-render of the component. This helps minimize re-rendering
+ * and makes the entire component more performant because it's not re-rendering when a bunch of properties change which aren't ever used in the UI.
+ * @param {Object} [report]
+ * @returns {Object|undefined}
+ */
+const reportSelector = report => report && ({
+    reportID: report.reportID,
+    participants: report.participants,
+    hasDraft: report.hasDraft,
+    isPinned: report.isPinned,
+    errorFields: {
+        addWorkspaceRoom: report.errorFields && report.errorFields.addWorkspaceRoom,
+    },
+    maxSequenceNumber: report.maxSequenceNumber,
+    lastReadSequenceNumber: report.lastReadSequenceNumber,
+    lastMessageText: report.lastMessageText,
+    lastMessageTimestamp: report.lastMessageTimestamp,
+    iouReportID: report.iouReportID,
+    hasOutstandingIOU: report.hasOutstandingIOU,
+    statusNum: report.statusNum,
+    stateNum: report.stateNum,
+    chatType: report.chatType,
+    policyID: report.policyID,
```

Comment:
Hey 👋, this PR caused [this regression](https://github.com/Expensify/App/issues/13457) where room names are not updated in the sidebar when renamed while offline. The problem is that `reportName` was not included in the reportSelector.